pages/login.tsx

```diff
@@ -20,26 +21,27 @@ type Values = {
   password: string
 }
 
-// TODO: Error Handling for login / signup. Blocked by backend implementation.
-const handleSubmit = async (values: Values) => {
-  const res = await fetch(`${SERVER_URL}/signin`, {
-    method: 'POST',
-    headers: {
-      'content-type': 'application/json'
-    },
-    body: JSON.stringify({
-      username: values.username,
-      password: values.password
+const Login: React.FC<Props> = () => {
+  const [invalidCredentials, setInvalidCredentials] = useState(false)
+
+  // TODO: Error Handling for login / signup. Blocked by backend implementation.
+  const handleSubmit = async (values: Values) => {
+    const res = await fetch(`${SERVER_URL}/signin`, {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json'
+      },
+      body: JSON.stringify({
+        username: values.username,
+        password: values.password
+      })
     })
-  })
-  const data = await res.json()
-  if (data.success) {
+    if (res.redirected) {
+      setInvalidCredentials(true)
```

Comment:
if I read `setInvalidCredentials(true) ` I'm expecting you are doing some kind of logic but not to display a message.